tests/e2e/swarm_test.go

```diff
@@ -112,7 +114,7 @@ func (s *SwarmTestSuite) TestDockerSwarm(c *C) {
 	status, host := verification.IsDockerContainerRunning(s.swarmNodes, s.serviceName, 2)
 	c.Assert(status, Equals, true, Commentf("Container %s is not running on any hosts", s.serviceName))
 
-	containerName, err = dockercli.GetContainerName(host, s.serviceName+".1")
+	containerName, err := dockercli.GetContainerName(host, s.serviceName+".1")
```

Comment:
@shaominchen I did run into this issue. 

CI: https://ci.vmware.run/vmware/docker-volume-vsphere/362

```
# github.com/vmware/docker-volume-vsphere/tests/e2e
../tests/e2e/swarm_test.go:115: undefined: containerName
../tests/e2e/swarm_test.go:116: undefined: containerName
../tests/e2e/swarm_test.go:117: undefined: containerName
FAIL	github.com/vmware/docker-volume-vsphere/tests/e2e [build failed]
make[1]: *** [e2e-dkrVolDriver-test] Error 2
make: *** [e2e-dkrVolDriver-test] Error 2
```